plugins/auth-backend/src/identity/TokenFactory.ts

```diff
@@ -115,16 +115,16 @@ export class TokenFactory implements TokenIssuer {
   }
 
   private async getKey(): Promise<JSONWebKey> {
-    // Make sure that we only generate one key at a time
+    // Make sure that we only generate one key at a time¨
     if (this.privateKeyPromise) {
-      if (this.keyExpiry?.isAfter()) {
+      if (this.keyExpiry > DateTime.local()) {
```

Comment:
Since, `this.keyExpiry` could be `undefined` looking at the existing code?
```suggestion
      if (this.keyExpiry && this.keyExpiry > DateTime.local()) {
```